MC/config/PWGLF/ini/GeneratorLFHyperNucleiPbPbGap.ini

```diff
@@ -0,0 +1,7 @@
+[GeneratorExternal]
+fileName=${O2DPG_ROOT}/MC/config/PWGLF/pythia8/generator_pythia8_longlived.C
```

Comment:
This also seems to be the wrong filename still, right?

Reply:
Yes, your are right